latest/ug/automode/auto-accelerated.adoc

```diff
@@ -0,0 +1,342 @@
+
+----
+kubectl delete -f nodepool-gpu.yaml
+kubectl delete -f pod.yaml
+----
+
+'''
+
+Files:
+
+nodepool-gpu.yaml
+
+----
+apiVersion: karpenter.sh/v1
+kind: NodePool
+metadata:
+  name: gpu
+spec:
+  disruption:
+    budgets:
+    - nodes: 10%
+    consolidateAfter: 1h
+    consolidationPolicy: WhenEmpty
+  template:
+    metadata: {}
+    spec:
+      nodeClassRef:
+        group: eks.amazonaws.com
+        kind: NodeClass
+        name: default
+      requirements:
+        - key: "karpenter.sh/capacity-type"
+          operator: In
+          values: ["on-demand"]
+        - key: "kubernetes.io/arch"
+          operator: In
+          values: ["amd64"]
+        - key: "eks.amazonaws.com/instance-family"
+          operator: In
+          values:
+          - g6e
+          - g6
+      terminationGracePeriod: 24h0m0s
+----
+
+workload.yaml
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nvidia-smi
+spec:
+  nodeSelector:
+    eks.amazonaws.com/instance-gpu-memory: "45776"
+    eks.amazonaws.com/compute-type: auto
+  restartPolicy: OnFailure
+  containers:
+  - name: nvidia-smi
+    image: nvidia/cuda:12.8.0-cudnn-devel-ubuntu20.04
+    args:
+    - "nvidia-smi"
+    resources:
+      requests:
+        memory: "30Gi"
+        cpu: "3500m"
+        nvidia.com/gpu: 1
+      limits:
+        memory: "30Gi"
+        nvidia.com/gpu: 1
+----
+
+
+To remove all objects created, simply run the following commands:
+
+----
+kubectl delete -f .
+----
+
+== Example NodePools Reference
+
+=== Create an NVIDIA NodePool
+
+The following NodePool defines:
+
+* Only launch instances of `g6e` and `g6` family
+* Consolidate nodes when empty for 1 hour
+** The 1 hour value for `consolodateAfter` supports spiky workloads and reduce node churn. You can tune `consolidateAfter` based on your workload requirements.
+
+*Example NodePool with GPU instance family and consolidation*
+
+[source,yaml]
+----
+apiVersion: karpenter.sh/v1
+kind: NodePool
+metadata:
+  name: gpu
+spec:
+  disruption:
+    budgets:
+    - nodes: 10%
+    consolidateAfter: 1h
+    consolidationPolicy: WhenEmpty
+  template:
+    metadata: {}
+    spec:
+      nodeClassRef:
+        group: eks.amazonaws.com
+        kind: NodeClass
+        name: default
+      requirements:
+        - key: "karpenter.sh/capacity-type"
+          operator: In
+          values: ["on-demand"]
+        - key: "kubernetes.io/arch"
+          operator: In
+          values: ["amd64"]
+        - key: "eks.amazonaws.com/instance-family"
+          operator: In
+          values:
+          - g6e
+          - g6
+      terminationGracePeriod: 24h0m0s
+----
+
+In addition, GPU workloads might be sensitive to specific GPU memory requirements, you might set a `eks.amazonaws.com/instance-gpu-memory` node selector on workloads so they schedule on nodes that have the required GPU memory resources. For other well-known labels which influence scheduling review, see <<auto-supported-labels>>.
```

Comment:
see comment above - customers typically choose based off the GPU card type due to the NVIDIA compute capabilities of that card. this can be done by either specifying the instance family types or using something like `karpenter.k8s.aws/instance-gpu-name`

Reply:
Suggestion:
"In addition, GPU workloads might be sensitive to specific GPU memory requirements, you might set a `eks.amazonaws.com/instance-gpu-memory` node selector along side `eks.amazonaws.com/instance-gpu-name` or `eks.amazonaws.com/instance-family` selector so they schedule on nodes that have the required GPU and resources. For other well-known labels which influence scheduling review, see <<auto-supported-labels>>"